src/users/userController.js

```diff
@@ -48,7 +50,7 @@ const userLogin = async (req, res, next) => {
     const { password, email } = req.body;
 
     const user = await userModel.findOne({ email });
-    if (!user) {
+    if (!user || user.status !== "Verified") {
```

Comment:
if (user.status !== "Verified") {
   return res.status(401).send({ message: "User not verified" });
}